docs/contributing.md

```diff
@@ -0,0 +1,13 @@
+---
+id: contributing
+title: Contributing
+---
+
+# Contribution Guidelines
```

Comment:
This was an internal list, I don't think it's well suited for our docs website. Maybe we can have it in some form in the GitHub repo but I don't think it brings much in the website itself.